app/assets/stylesheets/article-show.scss

```diff
@@ -6,7 +6,7 @@
   height: 10px;
 }
 
-body.default-navbar-config {
```

Comment:
I renamed this config class for sticky/static header so it's more sensible naming for non DEV usage.